docs/user-guides/acme-dns-validation.md

```diff
@@ -0,0 +1,94 @@
+# Issuing an ACME certificate using DNS validation
+
+`cert-manager` can be used to obtain certificates from a CA using the [ACME][1] protocol. The ACME protocol supports various challenges which are used to prove ownership of a domain so that a valid certificate can be issued for that domain. One such challenge is `dns-01`. With the DNS challenge, you prove you own a domain by proving you control domain's DNS records. You do this by creating a TXT record with specific content.
```

Comment:
`proving you control a domains DNS records`

Reply:
`You do this by...`
`This is done by...`

(else users may think they need to do something)

Reply:
good point